src/TLuaInterpreter.cpp

```diff
@@ -2468,7 +2470,7 @@ int TLuaInterpreter::getFont(lua_State* L)
             }
         }
     } else {
-        font = pHost->mDisplayFont.family();
+        font = mudlet::self()->mConsoleMap[pHost]->mUpperPane->fontInfo().family();
```

Comment:
As before, I think this is better as:

    font = pHost->mpConsole->mUpperPane->fontInfo().family();